src/build/roc/Builtin.roc

```diff
@@ -252,6 +252,8 @@ Builtin :: [].{
 			List.fold(list, Item.default(), |acc, elem| acc + elem)
 		}
 
+		to_str : List(U8) -> Try(Str, [BadUtf8({ index: U64, problem: Str.Utf8Problem }), ..others])
```

Comment:
I think `List.to_str` should be `List(a) -> Str where { a.to_str() -> Str }`. Utf8 to string should be in Str.